docs/amp.md

```diff
@@ -0,0 +1,21 @@
+# AMP
+
+Google's Accelerated Mobile Pages project takes your content and delivers it via their infrastructure optimised for mobile devices and slower internet connections.
+
+Web pages with AMP versions may also be highlighted in Google search results pages in a richer format than a regular search result listing.
+
+Activating AMP will automatically make AMP versions of your content available. You can choose which content types to generate AMP versions for through the CMS admin and also opt-out on a per-post basis.
+
+By default your standard theme will just work however _the default behaviour is to remove any scripts_ so you will need to ensure your site degrades gracefully without JavaScript enabled.
+
+The AMP component is provided by the [AMP for WP plugin](https://github.com/humanmade/amp-wp), full [documentation for the plugin can be found here](https://amp-wp.org/documentation/getting-started/).
+
+## Native mode vs Paired mode vs Classic mode
+
+**Native mode** will override your default content URLs with the AMP version of the page. Validation errors are fixed automatically.
```

Comment:
How does one switch between these modes?

Reply:
Via the admin - no reason we couldn't make this a config option though.